docs: app mode and cluster mode explained (#1041)
spectrocloud/librarium: content/docs/01-introduction.md

 
With a unique approach to managing multiple clusters, Palette gives IT teams complete control, visibility, and production-scale efficiencies to provide developers with highly-curated Kubernetes stacks and tools based on their specific needs, with granular governance and enterprise-grade security.
 
 
![Palette product highlevel overview](/docs_introduction_product-overview.png)
 
## What Makes Spectro Cloud Different?
 
### Full-Stack Management
Unlike rigid and prepackaged Kubernetes solutions, Palette allows users to construct flexible stacks from OS, Kubernetes, CNI, and CSI to additional add-on application services. As a result, the entire stack - not just the infrastructure - of Kubernetes is deployed, updated, and managed as one unit, without split responsibility from virtual machines, base OS, Kubernetes infra, and add-ons.
 
### End-to-End Declarative Lifecycle Management
Palette offers the most comprehensive profile-based management for Kubernetes, enabling teams to drive consistency, repeatability, and operational efficiency across multiple clusters in multiple environments, with comprehensive day 0 - day 2 management.
 
### Any Environment

> Palette has the richest coverage in supported environments, including public clouds: AWS, Azure, Google Cloud (both IaaS and managed K8s services EKS/AKS/GKE); data center: VMware, OpenStack; Bare metal: Canonical MaaS; Edge.

Author: Agree - we should break it out.

 
### Developers

Development teams will get the flexibility and freedom they are looking for to increase the speed of innovation, whether it is the cluster template with the add-on application services or choosing a Kubernetes version with integrations like logging, monitoring, and service mesh for your application development. They need not worry about Kubernetes configurations but focus on the stuff that matters.
 
### IT Operations and SREs
 
Declarative management makes life easier for IT teams, with consistency, repeatability, and all the enterprise-grade controls and governance they need - especially when moving to production [Cluster Profiles](/glossary-all#clusterprofile) enable them to define and re-use full-stack clusters and support them across the entire lifecycle without having to write scripts, as well as integrate with existing tools and methodologies.
 
 
### IT Executives
 
With an open and enterprise-grade platform, IT leaders can get peace of mind without being locked into proprietary orchestration technologies or one-size-fits-all solutions. This helps lower the total cost of ownership (TCO) and reduce operational risk.

# Next Steps

> Learn more about Palette and how it can improve your Kubernete experience but also for your organization. Try [Palette](https://console.spectrocloud.com/) for free today and experience Kubernetes unlike anything before. 

Author: ```suggestion
Learn more about Palette and how it can improve your Kubernetes experience and those in your organization. Try [Palette](https://console.spectrocloud.com/) for free today and experience a better way of working with Kubernetes. 
```
